cmd/creinit/template/workflow/porExampleDev/workflow_test.go.tpl

```diff
@@ -0,0 +1,250 @@
+package main
+
+import (
+	"context"
+	_ "embed"
+	"encoding/json"
+	"math/big"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/cre-sdk-go/capabilities/blockchain/evm"
+	evmmock "github.com/smartcontractkit/cre-sdk-go/capabilities/blockchain/evm/mock"
+	"github.com/smartcontractkit/cre-sdk-go/capabilities/networking/http"
+	httpmock "github.com/smartcontractkit/cre-sdk-go/capabilities/networking/http/mock"
+	"github.com/smartcontractkit/cre-sdk-go/capabilities/scheduler/cron"
+	"github.com/smartcontractkit/cre-sdk-go/cre/testutils"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	"{{projectName}}/contracts/evm/src/generated/balance_reader"
+	"{{projectName}}/contracts/evm/src/generated/ierc20"
+	"{{projectName}}/contracts/evm/src/generated/message_emitter"
+)
+
+var anyExecutionTime = time.Unix(1752514917, 0)
+
+func TestInitWorkflow(t *testing.T) {
+	config := makeTestConfig(t)
+	runtime := testutils.NewRuntime(t, map[string]string{})
```

Comment:
note, I updated this to take a map from namespace (a string type) to ID (string type) to string. For the empty ones, you can put nil. For the one that has a secret, we can put "" for the namespace and "SECRET_ADDRESS" as the ID with a value of "0x1234567890123456789012345678901234567890"

@cedric-cordenier, can secrets exist with the blank namespace, or should we update our examples?